src/theory/booleans/proof_checker.cpp

```diff
@@ -214,7 +339,7 @@ Node BoolProofRuleChecker::checkInternal(PfRule id,
       // non-unit clauses will not occur themselves in their elimination sets.
       // If they do then they must be unit.
       std::vector<Node> lits;
-      if (children[i].getKind() == kind::OR && elim.count(children[i]) == 0)
+      if (children[i].getKind() == kind::OR && !elim.count(children[i]))
```

Comment:
This introduces an implicit cast from `std::size_t` to bool... I'd rather stay with `== 0`.
```suggestion
      if (children[i].getKind() == kind::OR && elim.count(children[i]) == 0)
```